wandb/sdk/launch/errors.py

```diff
@@ -0,0 +1,25 @@
+from wandb.errors import Error
+
+
+class LaunchError(Error):
+    """Raised when a known error occurs in wandb launch."""
+
+    pass
+
+
+class LaunchDockerError(Error):
+    """Raised when Docker daemon is not running."""
+
+    pass
+
+
+class ExecutionError(Error):
+    """Generic execution exception."""
+
+    pass
+
+
+class SweepError(Error):
```

Comment:
Is this unused?

Reply:
This does appear unused but I haven't added it, I just moved it